token/js/src/instructions/initializeTransferFee.ts

```diff
@@ -0,0 +1,90 @@
+import { struct, u8 } from '@solana/buffer-layout';
+import { publicKey } from '@solana/buffer-layout-utils';
+import { AccountMeta, PublicKey, TransactionInstruction } from '@solana/web3.js';
+import {
+    TokenInvalidInstructionDataError,
+    TokenInvalidInstructionKeysError,
+    TokenInvalidInstructionProgramError,
+    TokenInvalidInstructionTypeError,
+} from '../errors';
+import { TokenInstruction } from './types';
+
+/** TODO: docs */
+
+export interface InitializeTransferFeeAmountInstructionData {
+    instruction: TokenInstruction.TransferFeeExtension;
+    transferFeeAmount: Number;
+}
```

Comment:
There's no instruction that does this unfortunately -- perhaps you meant to map `InitializeTransferFeeConfig`? https://github.com/solana-labs/solana-program-library/blob/24baf875e9e19c26d694d28c557d33848c3a9180/token/program-2022/src/extension/transfer_fee/instruction.rs#L28